control-station/src/components/Dynamics/Dynamics.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Dynamics() {
+	return <div>Dynamics</div>;
+}
```

Comment:
For consistency with other component files, let's put the `export default` at the end of the file.